lib/themes/dosomething/paraneue_dosomething/scss/patterns/_cta.scss

```diff
@@ -50,48 +50,103 @@
       @include clearfix;
       display: table;
       padding: $base-spacing / 2 0;
+    }
 
-      @include media($medium) {
-        @include span(12);
-        padding: $base-spacing / 2 0;
+    // Layout for campaigns with a scholarship.
+    &.with-scholarship {
+      .cta__action {
+        width: auto;
+
+        @include media($larger) {
+          @include span(6 of 12 no-gutters);
```

Comment:
@DFurnes I needed to get the button flush on the left side and the padding was breaking that. If there is a better way, let me know! Still getting used to susy.